tpp/src/main/Member.js

```diff
@@ -483,4 +486,42 @@ export default class Member extends CoreMember {
             await this._client.verifyEidas(payload, signature);
         });
     }
+
+    /**
+     * Get url to bank authorization page for a token request.
+     *
+     * @param bankId {string} Bank Id
+     * @param tokenRequestId {string} Token Request Id
+     * @returns {GetBankAuthUrlResponse} url
+     */
+    getBankAuthUrl(bankId: string, tokenRequestId: string): Promise<GetBankAuthUrlResponse> {
```

Comment:
The purpose of this call is to just return the url (string). Just return a string - we usually try to hide the "request/response" logic from the Member.js layer anyway, especially if you only need to return a single string.